scripts/release/build/deb/package.sh

```diff
@@ -27,7 +27,13 @@ for pkg_name in "${PKG_NAMES[@]}"; do
         exit 1
     fi
 
-    cp -p "${DEBTMP}"/*.deb "${PKG_ROOT}/${pkg_name}_${CHANNEL}_${OS}-${ARCH}_${FULLVERSION}.deb"
+    if [[ "$pkg_name" =~ devtools ]]; then
+        BASE_NAME="algorand-devtools"
+    else
+        BASE_NAME=algorand
```

Comment:
should this be quoted

Reply:
I generally don't quote unless it needs variable expansion or preserving special chars.